.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Validate Version bump upon release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  validate-version:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Validate version in setup.py
+      run: |
+        VERSION=$(python setup.py --version)
+        echo "Version in setup.py: $VERSION"
+        if [[ "$VERSION" != "$GITHUB_REF_NAME" ]]; then
```

Comment:
This means that the release format will change from **v1.8.1** to **1.8.1** ?

Reply:
nice catch! I will add the `v` to the $VERSION variable

Reply:
fixed @moustaphacheikh  :)